mobile/ios/device/ios-proxy-services.ts

```diff
@@ -482,4 +482,8 @@ export class IOSSyslog {
 		};
 		this.plistService.readSystemLog(printData);
 	}
+
+	public close(): void {
+		this.plistService.close();
```

Comment:
On a side note we'd have to think things through should we decide to proceed with the `null` assignment.
For example is it possible for `read` to be invoked after `close` 💭